exdb/tests/browser_tests.py

```diff
@@ -217,7 +217,11 @@ def test_load(self):
 
 
 class HallStaffDashboardBrowserTest(DefaultLiveServerTestCase):
-
+    """
+    TODO: Once we have a login page put this back in...
```

Comment:
Out of curiosity, is there a difference between a line comment and a block comment to the linter?